### PR TITLE
DPTS-893 Add integration tests for service types and more 

### PR DESCRIPTION
**Pull request by timrickwood**

### CODE REVIEW

File: build/automation/var/profile/integration.mk [hunk 1]
```diff
@@ -22,7 +22,7 @@ TF_VAR_deployment_secrets := $(DEPLOYMENT_SECRETS)
 TF_VAR_image_version := $(or $(BUILD_TAG), latest)
 TF_VAR_s3_tf_state_key := $(PROJECT_ID)/$(ENV)/s3/terraform.state
 TF_VAR_security_groups_tf_state_key := $(PROJECT_ID)/$(ENV)/security-groups/terraform.state
-TF_VAR_core_dos_db_sg := sg-05fdf44634b750fbd
+# TF_VAR_core_dos_db_sg := sg-05fdf44634b750fbd
```
Comment (timrickwood) [inline build/automation/var/profile/integration.mk:25]: Will explain why this is changing and we can maybe remove after discussion

File: build/automation/var/profile/nonprod.mk [hunk 2]
```diff
@@ -22,8 +22,11 @@ TF_VAR_deployment_secrets := $(DEPLOYMENT_SECRETS)
 TF_VAR_image_version := $(or $(BUILD_TAG), latest)
 TF_VAR_s3_tf_state_key := $(PROJECT_ID)/$(ENV)/s3/terraform.state
 TF_VAR_security_groups_tf_state_key := $(PROJECT_ID)/$(ENV)/security-groups/terraform.state
-TF_VAR_core_dos_db_sg := sg-05fdf44634b750fbd
+# TF_VAR_core_dos_db_sg := sg-05fdf44634b750fbd
```
Comment (timrickwood) [inline build/automation/var/profile/nonprod.mk:25]: Ditto

File: infrastructure/stacks/security-groups/variables.tf [hunk 3]
```diff
@@ -10,6 +10,10 @@ variable "vpc_terraform_state_key" {
 # # SG
 # ##############
 
-variable "core_dos_db_sg" {
-  description = "The ID of the Core DoS sg database"
+# variable "core_dos_db_sg" {
```
Comment (timrickwood) [inline infrastructure/stacks/security-groups/variables.tf:13]: Will explain as part of discussion and we can remove after

File: build/automation/var/profile/integration.mk (hunk 1 above)
Comment (tamaragoldschmidt) [inline build/automation/var/profile/integration.mk:25]: as discussed remove commented code

File: build/automation/var/profile/nonprod.mk (hunk 2 above)
Comment (tamaragoldschmidt) [inline build/automation/var/profile/nonprod.mk:25]: as discussed remove commented code

**Review by tamaragoldschmidt — APPROVED** [review]

**Review by tamaragoldschmidt — APPROVED** [review]

**Pull request merged**